spec/requests/api/service_requests_spec.rb

```diff
@@ -79,4 +79,120 @@ def expect_result_to_have_user_email(email)
       expect_result_to_have_user_email(@user.email)
     end
   end
+
+  context "authorization" do
+    it "is forbidden for a user without appropriate role" do
+      api_basic_authorize
```

Comment:
Where are we creating a user without the correct role?  I don't see role mentioned below for situations where we are NOT forbidden.  Is it really just admin/non-admin?